index.js

```diff
@@ -2,22 +2,19 @@
 const path = require('path');
 const karmaPlugins = ['karma-*'];
 const libPkg = require('./package.json');
+const isCI = require('is-ci');
 
 // dynamically require karma plugins from `videojs-generate-karma-config`
 // as karma-* only works on project local plugins.
-Object.keys(libPkg.dependencies).forEach(function(p) {
-  if ((/^karma-/).test(p)) {
-    karmaPlugins.push(require(p));
+Object.keys(libPkg.dependencies).forEach(function(pkgName) {
```

Comment:
support scoped packages. noticed this when I tried to add edge/edgium support but there are only scoped packages. I think this fix should stay.